packages/core/src/components/Tooltip.svelte

```diff
@@ -99,6 +134,14 @@
 >
   <div class="visahoi-tooltip-title">
     <b>{$activeMarker?.tooltip.title}</b>
+
+    <!--The trash icon is shown in the tooltip only isEditModeActive is set to true-->
+    {#if $isEditModeActive}
```

Comment:
I'd suggest to wrap the trash icon and the close icon into another div element, so that the 'justify-content: space-between' on the parent element so that the icons are still on the right side of the tooltip header. Then, add some space between the icons.